src/main/resources/db/migration/V5__drop_unique_constraints_add_org_manager.sql

```diff
@@ -6,4 +6,4 @@ ALTER TABLE organisation ADD COLUMN company_url varchar(512);
 
 ALTER TABLE organisation ADD COLUMN sra_id varchar(255);
 
-INSERT INTO PRD_ENUM (ENUM_CODE, ENUM_NAME, ENUM_TYPE, ENUM_DESC) VALUES (4, 'ORGANISATION_ADMIN', 'PRD_ROLE', 'Identifies the first user of an organisation');
```

Comment:
We should not be making changes to previous migration scripts (even adding removing white-space) as it will be harder to read the git history for when a change was introduced and might even adversely effect Flyway. Please revert this change.